src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -163,9 +167,30 @@ public Importer(final Config config, final FcrepoClient.FcrepoClientBuilder clie
         if (bagProfile == null) {
             this.bagItFileMap = null;
         } else {
-            final File bagDir = config.getBaseDirectory().getParentFile();
-            final Bag bag = verifyBag(bagDir.toPath());
-            configureBagItFileMap(bag);
+            try {
+                // load the bag profile
```

Comment:
Perhaps it might be a little cleaner to refactor the try / catch block into a new method "loadBagProfile()" ?  It might improve the readability.